gcp_variant_transforms/libs/bigquery_vcf_data_converter.py

```diff
@@ -299,6 +316,32 @@ def _get_variant_calls(self, variant_call_records):
       variant_calls.append(variant_call)
     return variant_calls
 
+  def _reconstruct_annotation_str(self, alt_base, annotation_id):
+    # type: (Dict[str, Any], str) -> List[str]
+    """Returns annotation strings.
+
+    The annotation str starts with `alt`, then followed by the annotation values
+    that are reconstructed in the same order as `annotation_names`. (e.g.,
+    'A|upstream_gene_variant|MODIFIER|PSMF1|||||').
+    """
+    if not self._annotation_id_to_annotation_names:
+      raise ValueError('No annotation names defined. The annotation string'
+                       'cannot be reconstructed since the order is not '
+                       'provided.')
+    annotation_str = []
+    for annotation_map in alt_base.get(annotation_id):
+      annotation_values = [
+          alt_base.get(bigquery_util.ColumnKeyConstants.ALTERNATE_BASES_ALT)]
+      for annotation_name in self._annotation_id_to_annotation_names.get(
+          annotation_id):
+        value = annotation_map.get(annotation_name)
+        if not value and value != 0:
```

Comment:
I think just doing `if value is None` is easier to understand here (everything else should be taken care of by `str(value)` below).